lib/posix-process/Makefile.uk

```diff
@@ -6,23 +6,30 @@ LIBPOSIX_PROCESS_COMMON_INCLUDES-y     += -I$(LIBPOSIX_PROCESS_BASE)/musl-import
 CINCLUDES-$(CONFIG_LIBPOSIX_PROCESS)   += $(LIBPOSIX_PROCESS_COMMON_INCLUDES-y)
 CXXINCLUDES-$(CONFIG_LIBPOSIX_PROCESS) += $(LIBPOSIX_PROCESS_COMMON_INCLUDES-y)
 
-LIBPOSIX_PROCESS_SUPPRESS_FLAGS-y   += -Wno-unused-parameter
-LIBPOSIX_PROCESS_CFLAGS-y           += $(LIBPOSIX_PROCESS_SUPPRESS_FLAGS-y)
+LIBPOSIX_PROCESS_CFLAGS-$(CONFIG_LIBPOSIX_PROCESS_DEBUG) += -DUK_DEBUG
 
+LIBPOSIX_PROCESS_SRCS-y += $(LIBPOSIX_PROCESS_BASE)/deprecated.c
 LIBPOSIX_PROCESS_SRCS-y += $(LIBPOSIX_PROCESS_BASE)/process.c
+LIBPOSIX_PROCESS_SRCS-y += $(LIBPOSIX_PROCESS_BASE)/wait.c
+LIBPOSIX_PROCESS_SRCS-y += $(LIBPOSIX_PROCESS_BASE)/signals.c
+LIBPOSIX_PROCESS_SRCS-$(CONFIG_LIBPOSIX_PROCESS_CLONE) += $(LIBPOSIX_PROCESS_BASE)/clone.c
+LIBPOSIX_PROCESS_SRCS-$(CONFIG_LIBPOSIX_PROCESS_CLONE) += $(LIBPOSIX_PROCESS_BASE)/clonetab.ld
+COMPFLAGS-$(CONFIG_LIBPOSIX_PROCESS_PIDS) += -fno-builtin-exit -fno-builtin-exit-group
```

Comment:
These flags are important to set globally (list of compiler flags, language independent: `COMPFLAGS-y`) because otherwise GCC is inlining a built-in for exit instead of our function.